README.md

```diff
@@ -56,12 +56,41 @@ git commit -n  # --no-verify
 
 ## CI/CD
 
-TBD
+GitHub actions are used to run all tests and scans as part of pull requests.
+
+Security scans are also run on a scheduled basis. Weekly for static code scans, and daily for dependency scans.
 
 ### Deployment
 
+#### Staging
+
+GitHub Actions are used to deploy to staging after any PR is merged into main.
+
+Branch protection rules prevent any direct pushes to `main` or PRs from being merged into main until all tests and scans pass.
+
+#### Production
+
 TBD
 
+### Configuring ENV variables in cloud.gov
```

Comment:
[praise] I really appreciate this documentation, because this was all stuff I would have had to track down otherwise, not being particularly experienced with cloud.gov